include/picongpu/plugins/openPMD/openPMDWriter.hpp

```diff
@@ -1297,7 +1305,6 @@ make sure that environment variable OPENPMD_BP_BACKEND is not set to ADIOS1.
                 // local offset + extent
                 const pmacc::Selection<simDim> localDomain = Environment<simDim>::get().SubGrid().getLocalDomain();
                 mThreadParams.cellDescription = m_cellDescription;
-                mThreadParams.currentStep = currentStep;
```

Comment:
This is the root of the bug, in `notify()` where we call this method `dumpData()' we call `intersectRangeWithWindow()` where the timestep is required. So we pointed always to the time-step of the last dump performed within this plugin instance.